NeoWeb/Views/Team/Index.cshtml

```diff
@@ -554,6 +554,13 @@
             <div class="card-text">NeoResearch Co-Founder</div>
         </div>
     </div>
+    <div class="card">
+        <img class="card-img-top" data-original="@Helper.CDN/images/team/thomas.jpg" />
+        <div class="card-body">
+            <div class="card-title">Thomas Lobker</div>
+            <div class="card-text">City of Zion Developer</div>
```

Comment:
Are you a NeoResearch Developer or City of Zion Developer?

Reply:
@chenzhitong I am with City of Zion, thank you